app/harvesters/idref/idref_harvester.py

```diff
@@ -66,7 +70,15 @@ async def fetch_results(self) -> AsyncGenerator[RawResult, None]:
                 )
         pending_queries = set()
         num_sudoc_waiting_queries = 0
-        async for doc in IdrefSparqlClient().fetch_publications(builder.build()):
+
+        tasks_gen = stream.combine.merge(
+            IdrefSparqlClient().fetch_uri_publications_open_edition(
+                builder.build_person_openedition_query()
+            ),
+            IdrefSparqlClient().fetch_publications(builder.build()),
+        )
+
+        async for doc in tasks_gen:
```

Comment:
Error message
`svp-harvester/app/harvesters/idref/idref_harvester.py:81: UserWarning: Streamer is iterated outside of its context`